index.html

```diff
@@ -84,5 +84,92 @@ <h1>
         </ul>
       </section>
     </main>
+    <section class="popup-window invisible"></section>
+    <section id="Portfolio" class="recent-work">
```

Comment:
- [ ]  [REQUIRED]Good job implementing the 'works' section, however, you are also required to implement the 'About Myself' section (placed after the projects and before the contact form) as in the [Project Requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md). Please implement that and ask for another review